gensim/test/test_fasttext_wrapper.py

```diff
@@ -30,8 +30,7 @@ def testfile():
 
 class TestFastText(unittest.TestCase):
     def setUp(self):
-        ft_home = os.environ.get('FT_HOME', None)
-        self.ft_path = os.path.join(ft_home, 'fasttext') if ft_home else None
+        self.ft_path = os.environ.get('FT_HOME', None)
```

Comment:
unrelated change

Reply:
small refactoring by the way. Looks a bit prettier.